rdr_service/model/genomics.py

```diff
@@ -1478,8 +1478,8 @@ class GenomicSampleSwap(Base):
     name = Column(String(255), nullable=False)
     explanation = Column(String(512))
     open_investigation = Column(SmallInteger, nullable=False, default=0)
-    open_investigation_date = Column(DateTime)
-    closed_investigation = Column(DateTime)
+    open_investigation_date = Column(DateTime, nullable=True)
+    closed_investigation = Column(SmallInteger, nullable=False, default=0)
```

Comment:
Are there any environments that have data in the sample swap table?

Reply:
Good question, no not @ this time, the import is coming soon.